Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:20.04
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y composer php-fpm php-cli php-gd php-xml php-apcu php-mbstring php-intl apache2 apache2-utils libfcgi0ldbl task-spooler
+RUN apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
Is this necessary?

Reply:
It’s not, but it shrinks the size of the image. The expectation is that you don’t interactively work with the image in the long term. If you’d make a change to the setup you’d do it in the Dockerfile, then rebuild the image again. But I’m happy to remove it if you’d prefer.